app/src/main/java/com/veriff/demo/base/VeriffFlowModel.kt

```diff
@@ -0,0 +1,22 @@
+package com.veriff.demo.base
+
+import com.veriff.demo.data.dataSources.sessionToken.SessionTokenDataSourceI
+import com.veriff.demo.data.dataSources.sessionToken.SessionTokenDataSourceI.Callback
+import com.veriff.demo.utils.qrCodeParser.QrCodeContentsParserI
+
+open class VeriffFlowModel(private val qrCodeContentsParser: QrCodeContentsParserI,
```

Comment:
Let's stop using this weird `...Model` naming and replace with something more appropriate like `...Repository`

Reply:
Currently this model is acting like a repository since its very simple. I think it is better to keep the model and introduce repository when requirement get more complicated. Is that okay?